software/cryptotvgen/cryptotvgen/prepare_libs.py

```diff
@@ -48,7 +46,7 @@ def get_latest_supercop_version_url(sc_version):
             url = sc_base_url + match.group(1)
             version = match.group(2)
             print(f'Latest version of SUPERCOP seems to be {version} avariable from {url}')
-        except Exception as e: #TODO
+        except Exception as e:
```

Comment:
I agree that the TODO tags with no text is not helpful for other developers at the moment (I'm very much to be blamed for being lazy and not add a descriptive text), but this is a WIP, and the intended TODOs have not been addressed so I think it wouldn't hurt to keep them there for now. We can clean up the code at a later stage when we're sure all the desired functionality is in and perfectly working.

Reply:
Added all the TODO that do not provide any information back